testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/XtextParametrizedRunner.java

```diff
@@ -175,7 +175,7 @@ private Object createTestUsingFieldInjection() throws Exception {
 						+ " @Parameter fields counted: " + annotatedFieldsByParameter.size()
 						+ ", available parameters: " + fParameters.length + ".");
 			}
-			Object testClassInstance = getTestClass().getJavaClass().newInstance();
+			Object testClassInstance = getTestClass().getJavaClass().getConstructor().newInstance();
```

Comment:
What's this change about?

Reply:
`getJavaClass().newInstance()` has been deprecated